src/Settings/Transport.cpp

```diff
@@ -22,6 +22,12 @@ void SimulationGenerator::DefineOptions_Transport(
     s->DefineSetting(mod + "/" + subname + "/type", "Type of transport model to use.", (int_t)OptionConstants::EQTERM_TRANSPORT_NONE);
 }
 
+void SimulationGenerator::DefineOptions_ConfinementTime( DREAM::Settings* s)
+{
+	s->DefineSetting("eqsys/tau_perp/tau_perp", "Type of perpendicular confinement time to use", (int_t)OptionConstants::CONF_TIME_BOHM);
+    s->DefineSetting("eqsys/tau_perp/mixed", "If we use a mixed combination of perpendicular confinement time", (int_t)0);
```

Comment:
again, we are not using a 'mix' we are taking a maximum between two laws.

Reply:
So how should I name the variable, because I don't have better name in mind

Reply:
Never mind I did not see you comment above

Reply:
fixed in current version. Thanks!